docs/updating/from_4.0/to_4.1.md

```diff
@@ -0,0 +1,117 @@
+---
+latest_tag: '4.1.1'
+---
+
+# Update from v4.0.x to v4.1
+
+This update procedure applies if you are using v4.0.latest.
+
+Go through the following steps to update to v4.1.
+
+## Update the app to v4.1
+
+First, run:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer require ibexa/content:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer require ibexa/experience:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer require ibexa/commerce:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+Continue with updating the app:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer recipes:install ibexa/content --force -v
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer recipes:install ibexa/experience --force -v
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer recipes:install ibexa/commerce --force -v
+    ```
+
+The `recipes:install` command installs new YAML configuration files. Look through the old YAML files and move your custom configuration to the relevant new files.
+
+Review the `bundles.php` and leave only third-party entires and entries added by the `recipes:install` command, that start with `Ibexa\Bundle`.
+
+## Update the database
+
+Apply the following database update script:
+
+### Ibexa DXP
+
+=== "MySQL"
+    ``` bash
+    mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-4.0.0-to-4.1.0.sql
+    ```
+
+=== "PostgreSQL"
+
+    ``` bash
+    psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-4.0.0-to-4.1.0.sql
+    ```
+
+## Configure the Product Catalog [[% include 'snippets/experience_badge.md' %]]
+
+Regardless of whether your application already uses the Product Catalog or you want to start using this functionality, you can choose to use old features, present in v4.0.x, or upgrade to the all new Product Catalog that v.4.1.x brings.
+
+Either way, make sure that in `config/packages/ibexa_shop.yaml` you set a value of the 
+`ibexa.commerce.site_access.config.eshop.default.catalog_data_provider` parameter correctly: 
+to use the old Product Catalog bundle, leave the value as-is and continue to the next section.
+To use the updated Product Catalog, change the value to `ibexa`, and perform the steps below.
```

Comment:
I'm pretty sure that after executing `composer recipes:install ibexa/experience --force -v` this statement is not longer a truth. Have you tests it?

Also please note that there is `config/packages/ibexa/shop.yaml` file is not shipped with product by default.